instrumentation/akka-http-10.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkahttp/client/HttpHeaderSetter.java

```diff
@@ -22,4 +29,26 @@ public void set(AkkaHttpHeaders carrier, String key, String value) {
           (HttpRequest) request.removeHeader(key).addHeader(RawHeader.create(key, value)));
     }
   }
+
+  public HttpRequest inject(HttpRequest original) {
+    AkkaHttpHeaders carrier = new AkkaHttpHeaders(original);
+    contextPropagators.getTextMapPropagator().inject(currentContext(), carrier, this);
```

Comment:
`Java8BytecodeBridge` is only needed inside of Advice, check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/writing-instrumentation-module.md

> You may have noticed that the example above does not use Context.current(), but a Java8BytecodeBridge method. This is intentional: if you are instrumenting a pre-Java 8 library, then inlining Java 8 default method calls (or static methods in an interface) into that library will result in a java.lang.VerifyError at runtime, since Java 8 default method invocations are not legal in Java 7 (and prior) bytecode. Because OpenTelemetry API has many common default/static interface methods (e.g. Span.current()), the javaagent-instrumentation-api artifact has a class Java8BytecodeBridge which provides static methods for accessing these default methods from advice. In fact, we suggest avoiding Java 8 language features in advice classes at all - sometimes you don't know what bytecode version is used by the instrumented class.

```suggestion
    contextPropagators.getTextMapPropagator().inject(Context.current(), carrier, this);
```